.github/workflows/container-tests.yml

```diff
@@ -105,6 +105,13 @@ jobs:
             branch: "master"
             tmt_repo: "https://gitlab.cee.redhat.com/platform-eng-core-services/sclorg-tmt-plans"
             tf_scope: "private"
+          - tmt_plan: "c8s"
```

Comment:
In case you want to test  Python 3.11, then this has to be filed as a separate pull request.

Reply:
Thank you, will do that.

Reply:
Done, merged and rebased.